fresh.py

```diff
@@ -174,7 +174,8 @@ def main():
     user = createUser()
 
     argparser = argparse.ArgumentParser()
-    argparser.add_argument("-s", "--sort", help="sort by hot or new", type=int)
+    argparser.add_argument("-s", "--sort", help="sort by hot, new, rising, random_rising, controversial or top", type=str,
+                           choices=['hot', 'new', 'rising', 'random_rising', 'controversial', 'top'])
```

Comment:
should this be here?

Reply:
@Ondkloss i was talking about line 178 where choices is indented, but obviously youve just shown that it runs fine :)